modules/javafx.controls/src/main/java/javafx/scene/control/ResizeFeaturesBase.java

```diff
@@ -31,6 +31,7 @@
 /**
  * A wrapper class for use by the column resize policies offered by
  * controls such as {@link TableView} and {@link TreeTableView}.
+ * @param <S> the type of the objects contained within the TableView items list
```

Comment:
Minor: This class isn't limited to TableView, so I recommend changing it to something like "the control's items list"